tracks/javascript/exercises/gigasecond/mentoring.md

```diff
@@ -0,0 +1,23 @@
+This exersize introduces [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).
+
+### Reasonable solutions
+```javascript
+// @ts-check
+
+const GIGASECOND_IN_MS = (10 ** 9) * 1000
+
+/**
+ * Adds a gigasecond (10^9) to the input {date}
+ *
+ * @export
+ * @param {Date} date
+ * @returns {Date} the date + 10^9 seconds
+ */
```

Comment:
remove typescript-related stuff

Reply:
That's not typescript, that is `JSDoc`.

Reply:
Ah ok, I don't use typescript and there was `@ts-enable` in previous commit which made me think this is typescript. 

Still I think we should not show docs in reasonable solutions - might lead to mentors recommending it.

Reply:
Ah. I personally hate the lack of documentation in both mentoring.md, solutions and javascript in the wild, but I'm happy to remove it!

Reply:
@tejasbubane what about this! I added a note about the JSDoc. I think adding it to _all_ our mentoring notes could help our mentors quickly see what the expected types are of input and output, since this is JS and not TS. I also agree that it should *NOT* be part of the solution as in not a suggestion to the student.

Reply:
Don't get me wrong - I am a huge fan of writing types of input-output especially after [I started learning Haskell](https://github.com/tejasbubane/haskell-book-code) where writing types for **every** function is a solid convention. But I am still not convinced about showing it in the expected solution for JS (maybe I am biased because I don't use JSdoc).

This applies to all other untyped languages as well. Mind opening an issue so we can discuss for others as well? eg [ruby](https://github.com/exercism/website-copy/blob/master/tracks/ruby/exercises/hamming/mentoring.md) and [elixir](https://github.com/exercism/website-copy/blob/master/tracks/elixir/exercises/binary-search/mentoring.md) don't show types but [Haskell](https://github.com/exercism/website-copy/blob/master/tracks/haskell/exercises/isogram/mentoring.md) obviously does.

Reply:
I'll open an issue, remove it for now -- can add it later after discussed.